packages/expect/__typetests__/expect.test.ts

```diff
@@ -14,94 +14,10 @@ import {
   type Matchers,
   type Tester,
   type TesterContext,
-  expect,
+  expect as _expect,
```

Comment:
Must rename, otherwise it collides with TSTyche's `expect`.

Reply:
`expect as jestExpect`?

Reply:
Could be. It just felt confusing, because in `@jest/expect` there is `jestExpect`, and `_expect`, and TSTyche's `expect`. So I was keeping `_expect` everywhere. Also could be `expectExpect`. Funny, but why not? ;D

Reply:
I think when reading the code, `jestExpect` is easier to follow. Especially more so than `_expect` which is usually meant as "I know this is unused, don't warn me about it" to eslint/typescript

Reply:
Ok. Let’s do it. This will be useful in `@jest/types` as well.